GETTING_STARTED_GUIDE.md

```diff
@@ -0,0 +1,153 @@
+## Getting started guide
+
+### How to run assistant on your own machine
+Below are the set of steps to run OpenSearch and OpenSearch dashboards with the OpenSearch assistant and the query generation functionality in the Observability Log Explorer page correctly on the cluster.
+**Note** that the `feature/langchain` is the branch used in this guide.
```

Comment:
Not sure if we should polish this GUIDE based on Agent framework implementation. @joshuali925